src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -521,6 +521,21 @@ export default class AircraftModel {
         return this.positionModel.relativePosition;
     }
 
+    /**
+     * Returns the appropriate string to use in
+     * row one of a data block.
+     *
+     * @property timeShareDestination
+     * @type {string}
+     */
+    get timeShareDestination() {
+        if (this.isArrival()) {
+            return this.destination.toUpperCase();
+        }
+
+        return this.fms.currentWaypoint.name.slice(0, 3);
```

Comment:
This should be constantly showing the departure procedure or first fix in the route. Even after passing the end of the SID, or that first fix, or whatever-- it should continue to show that as its value. So set it and forget it.

A `this.fms.getFullRouteString()` method will need to be added, and then we pull the SID exit (or when absent, the first fix) from that route string.

Reply:
Correct, it doesn't go here, but wherever the scratchpad is initialized for a departing aircraft is where this would be applied to get the desired behavior.

Currently, I think it initializes to the icao of destination airport, or `'XXX'` when the destination airport doesn't exist.